app/domain/authentication/authn_jwt/identity_providers/identity_from_decoded_token_provider.rb

```diff
@@ -2,21 +2,20 @@ module Authentication
   module AuthnJwt
     module IdentityProviders
       # Class for providing jwt identity from the decoded token from the field specified in a secret
-      class IdentityFromDecodedTokenProvider < IdentityProviderInterface
+      class IdentityFromDecodedTokenProvider
```

Comment:
can we separate this class into 2 command classes:
1. FetchJwtIdentity
2. ValidateIdentityConfiguredProperly

Let's try to avoid regular classes that perform actions. We should have either:
1. Command Class
2. ValueObject